uwblueprint/extend-a-family
# Add Help request model

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/7e51fde5f22d448b8226f2cfa4dd93d5?v=b2dc0dba06574651a8b6063bdd0d74e0&p=a82d483d0f564e0bb4104c4acc40c9f4&pm=s)



## Implementation description
* Implemented a new model for getting help with content
* 



## Steps to test
1. Look at the file "test_help_model.ts" and make any changes needed
2. Run it with ts-node ./test_help_model.ts in the backend folder
3. Check if the model exists in MongoDB



## What should reviewers focus on?
* Focus on the course help request portion, the facilitator and learner aspect is from Cherry's ticket and may change
* I stored the learner and facilitator as the object since they only container a little bit of extra info,
* For the unit, module and page, I stored them as the object id instead. I think this is better since if were to store them as the objects, we would have lots of duplicate information stored since CourseUnits include all Modules which includes all Pages and we only need to know the location of each individual part.


## Checklist
